system/jlib/jlog.ipp

```diff
@@ -837,6 +836,91 @@ private:
 #endif
 };
 
+class FieldLogAccessFilter : public CLogAccessFilter
```

Comment:
I'm not really sure what any of these classes are providing.  I think they provide a way of an xml tree for a filter, but why not define the xml representation and create it directly?